app/src/main/java/de/westnordost/streetcomplete/quests/smoothness/SmoothnessItem.kt

```diff
@@ -0,0 +1,161 @@
+package de.westnordost.streetcomplete.quests.smoothness
+
+import android.content.Context
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.smoothness.Smoothness.*
+import de.westnordost.streetcomplete.view.CharSequenceText
+import de.westnordost.streetcomplete.view.ResImage
+import de.westnordost.streetcomplete.view.ResText
+import de.westnordost.streetcomplete.view.image_select.DisplayItem
+import de.westnordost.streetcomplete.view.image_select.Item2
+
+fun Array<Smoothness>.toItems(context: Context, surface: String) =
+    mapNotNull { it.asItem(context, surface) }
+
+// return null if not a valid combination
+fun Smoothness.asItem(context: Context, surface: String): DisplayItem<Smoothness>? {
+    val imageResId = getImageResId(surface) ?: return null
+    val descriptionResId = getDescriptionResId(surface) ?: return null
+    return Item2(
+        this,
+        ResImage(imageResId),
+        CharSequenceText(context.getString(titleResId) + " " + emoji),
+        ResText(descriptionResId)
+    )
+}
+
+/** return fitting vehicle type emoji that corresponds to the "usable by" column in the wiki */
+val Smoothness.emoji get() = when(this) {
+    EXCELLENT ->     """🛹""" // or 🛼 but it is only available since Android 11
+    GOOD ->          """🛴""" // no emoji for racing bike, would be difficult to tell apart from 🚲
+    INTERMEDIATE ->  """🚲""" // or 🛵 but users are more likely to own a bike than a scooter
+    BAD ->           """🚗""" // or 🛺 but tuk-tuks have actually similar requirements as scooters
+    VERY_BAD ->      """🚙""" // this is a SUV
+    HORRIBLE ->      """🛻""" // no emoji for off-road vehicles but there is one for pick-ups (Android 11)
```

Comment:
Is it assured that the user will see emoji fonts that sufficiently distinguish between these vehicle types? I haven’t looked at this PR on an Android device yet, but some emoji fonts out there don’t meaningfully distinguish between U+1F697 and U+1F699 – they just recolor the same car body red or blue. There isn’t necessarily a tire hanging off the rear end of U+1F699, for example.

Reply:
On Android, generally the google emojis are used. The only emoji that is quite new and will not display on many devices is the pick-up emoji but this is not that bad because horrible surfaces and worse also mention in text that it is not eligible for vehicles that are not off-road capable

Reply:
> On Android, generally the google emojis are used.

An exception here is Samsung, according to https://emojipedia.org/google/.

Here is a comparison on how these emojis look like in different fonts:

* 🛹 https://emojipedia.org/skateboard/
* 🛴 https://emojipedia.org/kick-scooter/
* 🚲 https://emojipedia.org/bicycle/
* 🚗 https://emojipedia.org/automobile/
* 🚙 https://emojipedia.org/sport-utility-vehicle/
* 🛻 https://emojipedia.org/pickup-truck/
* 🚜 https://emojipedia.org/tractor/
* 🚶 https://emojipedia.org/person-walking/

And the alternative emojis in the code comments:

* 🛼 https://emojipedia.org/roller-skate/
* 🛵 https://emojipedia.org/motor-scooter/
* 🛺 https://emojipedia.org/auto-rickshaw/

Reply:
Unfortunately, Samsung depicts 🚗 as a sedan and 🚙 as a subcompact/supermini hatchback, which doesn’t suggest a difference in ruggedness.


![🚗](https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/samsung/312/automobile_1f697.png) ![🚙](https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/samsung/312/sport-utility-vehicle_1f699.png)